coast/COAsT.py

```diff
@@ -4,9 +4,13 @@
 from dask.distributed import Client
 
 
+def setup_dask_clinet(workers=2, threads=2, memory_limit_per_worker='2GB'):
```

Comment:
This creates a Client object but doesn't do anything with it?

Reply:
No - so there are some issues with the daks client and PyCharm - we don't need to access the client currently so this is more of a future helping method. 

The client is also somewhat detached from dask, as in you don't need to pass it around once build - in fact not calling the client only cause dask to use all available resources to hand ;)